### ISSUE TITLE
Limit lines, overflow

### ISSUE THREAD

**Issue by mandrachek** [mikepenz/multiplatform-markdown-renderer#246]
I've got a situation where I need to display a small portion of text, say just the first couple of lines from a couple of paragraphs (that might contain markdown). I'd normally do this with a `Text(maxLines=2, overflow = TextOverflow.Ellipsis)` - but this currently isn't possible with `Markdown()`.  

Since the content is just text, I can display the raw text, without the markdown formatting applied - but it would be a nice to have to be able to apply these limits also _after_ the markdown formatting has been applied, to be able to present the formatting present in these "snippets".

**Comment by mikepenz** [comment]
As the `Markdown` component itself is not designed to be used in such a way (each element is its own component, so the definition of 2 lines does not really apply) this is not possible in such a way. 

However, as an alternative I've reworked the internal annotated string builder and added a new util function you can use to achieve your requirement. 

Here's a small snippet: 

```kotlin
Text(
                    text = "This is a paragraph with some *italic* and **bold** text. This is a paragraph with some *italic* and **bold** text. This is a paragraph with some *italic* and **bold** text.".buildMarkdownAnnotatedString(MaterialTheme.typography.body1),
                    maxLines = 2
)
```

**Comment by mikepenz** [comment]
Available in [v0.28.0-rc01](https://github.com/mikepenz/multiplatform-markdown-renderer/tree/refs/tags/v0.28.0-rc01)
